README.md

```diff
@@ -1,102 +1,170 @@
+    <td>Installs or removes the te-agent package</td>
+    <td><tt>true</tt></td>
+  </tr>
+  <tr>
+    <td><tt>agent_service_enable</tt></td>
+    <td>Boolean</td>
+    <td>Enables or disables the te-agent service</td>
+    <td><tt>*equals agent parameter value*</tt></td>
+  </tr>
+  <tr>
+    <td><tt>browserbot</tt></td>
+    <td>Boolean</td>
+    <td>Installs or removes Browserbot</td>
+    <td><tt>false</tt></td>
+  </tr>
+  <tr>
+    <td><tt>set_repository</tt></td>
+    <td>Boolean</td>
+    <td>Includes or not the ThousandEyes repository</td>
+    <td><tt>true</tt></td>
+  </tr>
+  <tr>
+    <td><tt>agent_utils</tt></td>
+    <td>Boolean</td>
+    <td>Installs or removes the agent utilities</td>
+    <td><tt>false</tt></td>
+  </tr>
+  <tr>
+    <td><tt>international_langs</tt></td>
+    <td>Boolean</td>
+    <td>Installs or removes the language package</td>
+    <td><tt>false</tt></td>
+  </tr>
+  <tr>
+    <td><tt>account_token</tt></td>
+    <td>String</td>
+    <td>Account token for the agent</td>
+    <td><tt>'account_token' (Sample value)</tt></td>
+  </tr>
+  <tr>
+    <td><tt>log_path</tt></td>
+    <td>String</td>
+    <td>Agent log path</td>
+    <td><tt>'/var/log'</tt></td>
+  </tr>
+  <tr>
+    <td><tt>log_level</tt></td>
+    <td>String</td>
+    <td>Log level</td>
+    <td><tt>'DEBUG'</tt></td>
+    <td><tt>'DEBUG','TRACE'</tt></td>
+  </tr>
+  <tr>
+    <td><tt>log_file_size</tt></td>
+    <td>Integer</td>
+    <td>Log file size (MB)</td>
+    <td><tt>10</tt></td>
+  </tr>
+  <tr>
+    <td><tt>num_log_files</tt></td>
+    <td>Integer</td>
+    <td>Amount of log files.</td>
+    <td><tt>13</tt></td>
+  </tr>
+  <tr>
+    <td><tt>proxy_location</tt></td>
+    <td>String</td>
+    <td>Proxy location (IP:PORT)</td>
+    <td><tt>-</tt></td>
+  </tr>
+  <tr>
+    <td><tt>proxy_user</tt></td>
+    <td>String</td>
+    <td>Proxy username</td>
+    <td><tt>-</tt></td>
+  </tr>
+  <tr>
+    <td><tt>proxy_pass</tt></td>
+    <td>String</td>
+    <td>Proxy password</td>
+    <td><tt>-</tt></td>
+  </tr>
+  <tr>
+    <td><tt>proxy_bypass_list</tt></td>
+    <td>String</td>
+    <td>Proxy bypass list (Comma separated values)</td>
+    <td><tt>-</tt></td>
+  </tr>
+  <tr>
+    <td><tt>proxy_type</tt></td>
+    <td>String</td>
+    <td>Proxy type</td>
+    <td><tt>'DIRECT'</tt></td>
+    <td><tt>'DIRECT','STATIC','PAC'</tt></td>
+  </tr>
+  <tr>
+    <td><tt>proxy_auth_type</tt></td>
+    <td>String</td>
+    <td>Proxy authentication type</td>
+    <td><tt>-</tt></td>
+    <td><tt>'BASIC','KERBEROS','NTLM'</tt></td>
+  </tr>
+  <tr>
+    <td><tt>crash_reports</tt></td>
+    <td>Integer</td>
+    <td>Enables or disables crash reports</td>
+    <td><tt>1</tt></td>
+    <td><tt>0, 1</tt></td>
+  </tr>
+
+</table>
 
- ```
- class { 'teagent':
-     account_token       => 'your_account_token_goes_here',
-     browserbot          => true,
-     international_langs => true,
- }
- ```
-
- * Set the log path location
 
+ * Default settings
```

Comment:
This `Default settings` and `Set the proxy...` i think they would be better before the above table. Otherwise someone seeing the table without an example would be a bit out of context.